### DISCUSSION TITLE
Running GoCD agent on windows as a user

### DISCUSSION THREAD

**Discussion by pradeep19980** [gocd/gocd#12066] [Q&A]
I created a user on Windows and downloaded the GoCD agent using that user account. However, when I trigger a pipeline, it uses 'nt authority\system' to run GoCD instead of using the desired user account. Do you have any suggestions for resolving this issue?

The command I used to download is choco install gocdagent --version=20.10 --ia "/S /START_AGENT=NO /D=C:\Users\GoCD\Go"

**Comment by chadlwilson** [comment]
Change the user the service is running with via Services. You'll need to ensure the user has Full access to the the agent install location, assuming you are not overwriting the various log, data and temp directories it needs.

**Reply by pradeep19980** [reply]
I have tried this and set the service to use the user I specified but when I trigger pipeline it chooses NT authority

**Reply by chadlwilson** [reply]
It sounds like you might not have done it correctly. Are you sure the agent java process is running with the correct user in task manager?
